tests/e2e/specs/shopper/mini-cart.test.js

```diff
@@ -218,4 +219,41 @@ describe( 'Shopper → Mini Cart', () => {
 			);
 		} );
 	} );
+
+	describe( 'Cart page', () => {
+		beforeAll( async () => {
+			await shopper.emptyCart();
+		} );
+
+		it( 'Can go to cart page from the Mini Cart Footer', async () => {
+			const productTitle = await page.$eval(
```

Comment:
Just checking what does `$$eval`, I found this utility in our codebase `getTextContent` (https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/f79618fe252ebe4ed97f6059e6fd81c149dd64aa/tests/e2e/page-utils.js#L53). I guess that we should use it

Reply:
Yes. Thanks for the head-up! I'm already using it in #5979 but forgot to update this one!